grafana-plugin/src/containers/UserSettings/parts/tabs/GoogleCalendar/GoogleCalendar.tsx

```diff
@@ -15,6 +15,27 @@ import { UserHelper } from 'models/user/user.helpers';
 import { ApiSchemas } from 'network/oncall-api/api.types';
 import { useStore } from 'state/useStore';
 import { UserActions } from 'utils/authorization/authorization';
+import { DOCS_ROOT } from 'utils/consts';
+
+const Heading: React.FC<{ connected: boolean }> = ({ connected }) => (
+  <HorizontalGroup>
+    <GoogleCalendarLogo width={32} height={32} />
+    <div>
+      <Text.Title level={5}>{connected ? 'Google calendar is connected' : 'Connect your Google Calendar'}</Text.Title>
+      <Text type="secondary">
+        This connection allows Grafana OnCall to read your Out of Office events and autogenerate Shift Swap Requests (
+        <a
+          href={`${DOCS_ROOT}/manage/on-call-schedules/shift-swaps/#google-calendar-integration`}
+          target="_blank"
+          rel="noreferrer"
+        >
+          <Text type="link">docs</Text>
+        </a>
+        ).
+      </Text>
+    </div>
+  </HorizontalGroup>
```

Comment:
(simply adds the `(docs)` link):

<img width="1096" alt="Screenshot 2024-04-15 at 14 43 58" src="https://github.com/grafana/oncall/assets/9406895/179e22f3-d870-4ae4-b484-50c480989ad7">